db/migrations/20190522150559_users.js

```diff
@@ -7,6 +7,8 @@ exports.up = function(knex, Promise) {
       email VARCHAR(100) UNIQUE NOT NULL,
       avatar VARCHAR(100),
       password VARCHAR(100),
+      reset_token VARCHAR(100),
```

Comment:
Is this pre-existing migration? If so, we can't just add new columns to already executed migrations, we need a new one.

Also, reconsider having `reset_toket` and `reset_token_expiry` in a separate `login_sessions` table instead of `users` table.